### DISCUSSION TITLE
Missing Checkmark/Tickbox Option for Habit

### DISCUSSION THREAD

**Discussion by MechtEngineer** [iSoron/uhabits#1105] [Help & FAQ] [answered]
The new ability in v2.0.3 to put a number against the habit achieved on a daily basis is a great feature enhancement, but for habits that only need to be completed once per day, or multiple times per week (but still once per day), the new method of having to enter a "1" for these types of habits is _very_ inefficient compared to the simple toggling of a checkmark on and off. Would it be possible to designate the type of habit (binary or numerical) so the user can use the default efficient checkmark option, and change the habit to record numbers when desired?

Thanks for developing such an amazing app!

**Comment by KristianTashkov** [comment, marked as answer]
Hello @MechtEngineer!
You already have this options when you start creating a new habit. It asks you whether you want a "YES/NO" type habit or a "Measurable" type habit. Are you having problems creating "YES/NO" habits?

**Reply by MechtEngineer** [reply]
Thanks @KristianTashkov, that was my problem. The way those two questions appeared on my screen made me just think it was an "information" screen for newbies. I misinterpreted them! Maybe a user interface cue might help others in the future? Some ideas are:

- a "heading" to say they are the type of habit the user is creating (e.g. "Habit Options"
- an outline around each option in a color, to show that they are buttons and not just information. I use the "dark mode" with black being black. Currently there is no distinction between the two options, hence why I thought it was an "information screen", and just pressed the text randomly to setup the habit. It turns out they were "buttons"!

**Reply by MechtEngineer** [reply]
By the way, I think the new feature is great! I think the app is utterly splendid too! :)

**Reply by KristianTashkov** [reply]
Now that I look at it with fresh eyes, you make a good point that these don't really look like buttons on a pure black theme. I will try to propose a fix during the weekend for this if I find the time!

**Reply by MechtEngineer** [reply]
Thanks, that would be awesome!
